src/system/sargon_os/sargon_os_profile.rs

```diff
@@ -161,6 +127,10 @@ impl SargonOS {
         Ok(profile_id)
     }
 
+    pub fn profile(&self) -> Profile {
```

Comment:
preserve comment:

```rust
    /// Returns the current profile in full. This is a COSTLY operation
    /// and hosts SHOULD NOT do it lightheartedly, prefer using more specific
    /// reading operations such as `os.current_network_id` or `os.accounts_for_display_on_current_network` etc, which are cheap operations compared
    /// to using this.
    ///
    /// In the future will will most likely deprecate this method.
```
? 

I think it important to keep it documented that this is in fact a very costly UniFFI call.

Reply:
Sure, this will go away completely anyway with ProfileState that will be introduced by @micbakos-rdx.